eth/src/main/kotlin/com/d3/eth/abi/AbiGsonHelper.kt

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright D3 Ledger, Inc. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.d3.eth.abi
+
+import com.google.gson.*
+import java.lang.reflect.Type
+import javax.xml.bind.DatatypeConverter
+
+internal object AbiGsonHelper {
+    val customGson: Gson by lazy {
+        GsonBuilder().registerTypeHierarchyAdapter(
+            ByteArray::class.java,
+            ByteArrayToEthHexTypeAdapter()
+        ).create()
+    }
+
+    private class ByteArrayToEthHexTypeAdapter : JsonSerializer<ByteArray>, JsonDeserializer<ByteArray> {
+        @Throws(JsonParseException::class)
+        override fun deserialize(
+            json: JsonElement,
+            typeOfT: Type,
+            context: JsonDeserializationContext
+        ): ByteArray {
+            return DatatypeConverter.parseHexBinary(removePrefixIfNeeded(json.asString))
```

Comment:
```suggestion
            return org.web3j.utils.Numeric.hexStringToByteArray(json.asString)
```
Use this method instead, this allows you to remove custom methods for removing / adding a prefix